pages/pipelines/defining_steps.md.erb

```diff
@@ -1,52 +1,52 @@
 # Defining Your Pipeline Steps
 
-Pipeline steps can be defined visually in Buildkite or, more powerfully, in code using a `pipeline.yml` file.
+Pipeline steps are defined in YAML and are either stored in Buildkite or in your repository using a `pipeline.yml` file.
 
-Defining your pipeline steps in a `pipeline.yml` file gives you access to more configuration options than the web interface, and allows you to version, audit and review your build pipelines alongside your source code.
+Defining your pipeline steps in a `pipeline.yml` file gives you access to more configuration options and environment variables than the web interface, and allows you to version, audit and review your build pipelines alongside your source code.
 
 {:toc}
 
 ## Getting started
 
 Create a pipeline from the Pipelines page of Buildkite using the ➕ button.
 
-Required fields are 'name', 'repository', and 'steps'. You can leave the 'steps' with only the default empty command step to get started, the steps can be edited at any time from your pipeline settings page.
+Required fields are 'name', 'repository'.
 
 <%= image "new-pipeline-setup.png", width: 1768/2, height: 1169/2, alt: "Screenshot of the 'New Pipeline' setup form" %>
 
 You can set up webhooks at this point, but this step is optional. These webhook setup instructions can be found in pipeline settings on your specific repository provider page.
 
-You can also use the REST API to create a pipeline and define your steps. See the [Pipelines API](/docs/apis/rest-api/pipelines) for further information.
+Both the REST API and GraphQL API can be used to create a pipeline programmatically. See the [Pipelines REST API](/docs/apis/rest-api/pipelines) and the [GraphQL API](https://buildkite.com/docs/apis/graphql-api) for details and examples.
 
 ## Adding steps
 
-There are two ways to add steps to your pipeline: with the visual editor in Buildkite or with a `pipeline.yml` file. You can add steps immediately during your initial pipeline setup, or you can edit them on the pipeline settings page.
+There are two ways to define steps in your pipeline: using the YAML step editor in Buildkite or with a `pipeline.yml` file. The web steps visual editor is still available if you haven't migrated to YAML Steps, but is no longer recommended and will be deprecated in the future. 
```

Comment:
@toolmantim do you think this is enough to folks to see that the web editor is still around, or were you keen to have a section for it still? I've also added a link to the migration guide below this.

Reply:
```suggestion
There are two ways to define steps in your pipeline: using the YAML step editor in Buildkite or with a `pipeline.yml` file. The web steps visual editor is still available if you haven't migrated to [YAML Steps](https://buildkite.com/changelog/99-introducing-the-yaml-steps-editor) but will be deprecated in the future. 
```